### DISCUSSION TITLE
Is there a way to exclude tables from prisma generate?

### DISCUSSION THREAD

**Discussion by devalnor** [prisma/prisma#4440] [Q&A] [answered]
For example migration tables.
I have tried with an underscore like `_pgmigration` but it's not work.

Any idea or this is feature can be a request?

**Comment by ryands17** [comment, marked as answer]
Hey @devalnor 👋 

Unfortunately this isn't possible right now, and there's a similar request [here](https://github.com/prisma/prisma/issues/807), but if this still doesn't match your use case, I would suggest creating a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) :)

**Comment by devalnor** [comment]
Feature request: https://github.com/prisma/prisma/issues/4510

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
